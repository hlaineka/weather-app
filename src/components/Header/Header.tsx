import styled from 'styled-components';
import { Text } from '../Text/Text';
import { palette } from '../variables';

export const Header = () => {
    const content = "Säätutka";
    return (
        <HeaderContainer>
        <Text variant='header'>{content}</Text>
        </HeaderContainer>
    );

};

const HeaderContainer = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: center;
  margin: 0;
  max-width: 100%;
  width: 100%;
  background-color: white;
  border: 1px solid ${palette.greyBorder}
`;
