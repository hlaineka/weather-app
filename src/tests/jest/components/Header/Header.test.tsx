import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../../../components/Header';

describe('Header component', () => {
  test('should render component properly', () => {
    // when
    const componentRenderer = renderer.create(<Header />);
    const tree = componentRenderer.toJSON();

    // then
    expect(tree).toMatchSnapshot();
  });
});
