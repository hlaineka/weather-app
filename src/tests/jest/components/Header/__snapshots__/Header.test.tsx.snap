// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render component properly 1`] = `
<div
  className="sc-bczRLJ llmGys"
>
  <h1
    style={
      Object {
        "color": "#262626",
        "fontFamily": "Arial, sans-serif",
        "fontSize": "23pt",
        "fontStyle": "normal",
        "fontWeight": "400",
        "lineHeight": "23pt",
        "marginBottom": "23pt",
        "marginTop": "23pt",
      }
    }
  >
    Säätutka
  </h1>
</div>
`;
